index.bs

```diff
@@ -75,15 +75,18 @@ type: dfn
   transport or it isn't -- and encourage developers to securely load any mixed content that is
   necessary for their webpage to function properly.
 
-  Mixed Content Level 2 therefore updates and extends [[!mixed-content]] to provide users with
+  So this specification was updated to provide users with
   better security and privacy guarantees and a better security UX, while minimizing
-  breakage. Instead of advising browsers to simply strictly block all mixed content, the core idea
-  of Level 2 is <i>mixed content autoupgrading</i>. That is, mixed content that user agents are not
-  already blocking should be autoupgraded to a secure transport. If the request cannot be
-  autoupgraded, it will be blocked. Autoupgrading avoids loading insecure resources on secure
+  breakage. Among the updates made was — beyond the idea of just advising browsers to strictly
+  block all mixed content — the addition of the idea of <i>mixed content autoupgrading</i>:
```

Comment:
While I hate to break a good Spanish Inquisition reference, I think this'll be clearer without it:

```suggestion
  breakage. Instead of advising browsers to simply strictly block all mixed content, this specification advises <i>mixed content autoupgrading</i>:
```